packages/components/src/List/List.component.js

```diff
@@ -131,7 +138,7 @@ ListDisplay.propTypes = DisplayModeComponent.propTypes;
 }
  <List {...props}></List>
  */
-function List({ displayMode, id, list, toolbar, useContent, virtualized }) {
+export function List({ displayMode, id, list, toolbar, useContent, virtualized, t }) {
```

Comment:
Why are you exporting the non translated component ?
The translated one should behave the same as the non translated one if no configuration is provided.